### PR TITLE
The "keystorePath" configuration supports the built-in variable "projectDir"

### PR DESCRIPTION
**Pull request by beilly**

### CODE REVIEW

File: src/tools/uber-apk-signer.ts [hunk 1]
```diff
@@ -17,7 +17,8 @@ export namespace apkSigner {
         const extensionConfig =
             vscode.workspace.getConfiguration(extensionConfigName);
         const apkSignerPath = extensionConfig.get("apkSignerPath");
-        const keystorePath = extensionConfig.get("keystorePath");
+        const keystorePathTmp = extensionConfig.get("keystorePath");
+        const keystorePath = eval("`" + keystorePathTmp + "`");
```
Comment (Surendrajat) [inline src/tools/uber-apk-signer.ts:21]: @beilly thanks, but what exactly does it fix?

Comment (beilly) [reply src/tools/uber-apk-signer.ts:21]: The "keystorePath" configuration supports the built-in variable "projectDir"

image

Comment (Surendrajat) [reply src/tools/uber-apk-signer.ts:21]: So it will be the same for all other paths i guess, not just this one, right?

Comment (beilly) [reply src/tools/uber-apk-signer.ts:21]: Variables are not supported by default, right? This is done with local variables, and others can be done this way. Or if there is a better way, please let me know.

**Pull request closed without merging**